### PR TITLE
Feature release/bump minor version

### PR DESCRIPTION
**Pull request by yaojiefeng**

### CODE REVIEW

File: cdap-distributions/bin/build_docs_bucket.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ TARGET_DIR=${DOCS_HOME}/target
 
 S3_BUCKET=${S3_BUCKET:-docs.cask.co}
 S3_REPO_PATH=${S3_REPO_PATH:-cdap} # No leading or trailing slashes
-VERSION=${VERSION:-5.1.2}
+VERSION=${VERSION:-5.1.3-SNAPSHOT}
```
Comment (albertshau) [inline cdap-distributions/bin/build_docs_bucket.sh:47]: should just be 5.1.3 I think

File: cdap-distributions/src/emr/install.sh [hunk 2]
```diff
@@ -24,9 +24,9 @@ CDAP_BRANCH=${CDAP_BRANCH:-release/5.0}
 # Optional tag to checkout - All released versions of this script should set this
 # like this: CDAP_TAG=${CDAP_TAG:+tag} as this allows setting tag to empty/null
 # otherwise, it should be CDAP_TAG=''
-CDAP_TAG=${CDAP_TAG:+v5.1.2}
+CDAP_TAG=${CDAP_TAG:+v5.1.3-SNAPSHOT}
```
Comment (albertshau) [inline cdap-distributions/src/emr/install.sh]: should just be 5.1.3 I think

File: cdap-distributions/src/hdinsight/pkg/install.sh [hunk 3]
```diff
@@ -26,7 +26,7 @@ CDAP_BRANCH='release/5.0'
 # otherwise, it should be CDAP_TAG=''
 CDAP_TAG=${CDAP_TAG:+hdi5.0}
 # The CDAP package version passed to Chef
-CDAP_VERSION='5.1.2-1'
+CDAP_VERSION='5.1.3-SNAPSHOT-1'
```
Comment (albertshau) [inline cdap-distributions/src/hdinsight/pkg/install.sh]: should just be 5.1.3 I think

**Review by albertshau — COMMENTED** [review]
I believe the scripts shouldn't have snapshot in them (can check develop to make sure)

File: cdap-distributions/bin/build_docs_bucket.sh (hunk 1 above)
Comment (yaojiefeng) [reply cdap-distributions/bin/build_docs_bucket.sh:47]: On develop this is 6.0.0-SNAPSHOT, but the other two places is still 5.1.0, @dereklwood can you confirm which version we should use for the scripts?

File: cdap-distributions/src/emr/install.sh (hunk 2 above)
Comment (yaojiefeng) [reply cdap-distributions/src/emr/install.sh]: Confirmed this should be 5.1.3

File: cdap-distributions/src/hdinsight/pkg/install.sh (hunk 3 above)
Comment (yaojiefeng) [reply cdap-distributions/src/hdinsight/pkg/install.sh]: Confirmed this should be 5.1.3

File: cdap-distributions/bin/build_docs_bucket.sh (hunk 1 above)
Comment (yaojiefeng) [reply cdap-distributions/bin/build_docs_bucket.sh:47]: This is 6.0.0-SNAPSHOT on develop, so I will keep the snapshot here.

Comment (dereklwood) [reply cdap-distributions/bin/build_docs_bucket.sh:47]: I think this is the docs version, which should match the CDAP version (with SNAPSHOT), and shows up here:  https://docs.cdap.io/cdap/.  I think keeping the SNAPSHOT to stay consistent with CDAP  should be fine.

I suspect this script is not used anymore (not 100% sure though).  Bamboo Docs Deploy build uses the similar [docs_bucket_current_develop.sh](https://github.com/cdapio/cdap/blob/develop/cdap-distributions/bin/docs_bucket_current_develop.sh) which has logic to determine the version

Comment (dereklwood) [reply cdap-distributions/bin/build_docs_bucket.sh:47]: nvm, it is apparently still used in Docs Build and Test, Docs Repo stage

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**